### PR TITLE
Login Page integrated with react-hook-form

### PR DESCRIPTION
**Pull request by allie-rae**
https://user-images.githubusercontent.com/48700332/187065297-8fa2bf8d-a743-4801-a00a-90f4275d8718.mov

### CODE REVIEW

**Review by jae-finger — APPROVED** [review]
:D

File: src/pages/LoginPage/LoginPage.tsx [hunk 1]
```diff
@@ -1,9 +1,23 @@
...
 const LoginPage: React.FC = (): React.ReactElement => {
+  const { control, handleSubmit } = useForm<LoginInputs>({
+    mode: 'onBlur',
+    reValidateMode: 'onBlur',
+    defaultValues: {
+      password: '',
+      username: '',
+    },
+  });
   return (
```
Comment (zimashima) [inline src/pages/LoginPage/LoginPage.tsx:21]:
```suggestion
  
  return (
```

Comment (zimashima) [reply src/pages/LoginPage/LoginPage.tsx:21]: A blank line to separate the context :)

File: src/pages/LoginPage/LoginPage.tsx [hunk 2]
```diff
@@ -21,20 +35,70 @@ const LoginPage: React.FC = (): React.ReactElement => {
...
-        <TextField
-          label="Password"
-          type="password"
-          variant="outlined"
-          sx={{ mb: 2, width: '100%' }}
-        />
-        <Button variant="contained" fullWidth size="large" sx={{ mb: 2 }}>
-          Login
-        </Button>
+
```
Comment (zimashima) [inline src/pages/LoginPage/LoginPage.tsx:38]:
```suggestion
```

File: src/pages/LoginPage/LoginPage.tsx [hunk 3]
```diff
@@ -21,20 +35,70 @@ const LoginPage: React.FC = (): React.ReactElement => {
...
+                error={!!error && isTouched}
+                onChange={onChange}
+                onBlur={onBlur}
+                value={value}
+                name={name}
+              />
+            )}
+            rules={{ required: true }}
+          />
+
```
Comment (zimashima) [inline src/pages/LoginPage/LoginPage.tsx:65]:
```suggestion
```

**Pull request merged**